browser/net/ipfs_redirect_network_delegate_helper.cc

```diff
@@ -104,25 +109,19 @@ int OnHeadersReceived_IPFSRedirectWork(
   if (ctx->ipfs_auto_fallback && !api_gateway && response_headers &&
       response_headers->GetNormalizedHeader("x-ipfs-path", &ipfs_path) &&
```

Comment:
does `ipfs_path` still get used anywhere?

Reply:
It is used as a marker in this case